### ISSUE TITLE
[Bug]: Publii 0.44.2 show error when restore backup

### ISSUE THREAD

**Issue by Cineray** [GetPublii/Publii#1310]
### Operating system

Windows 11

### Publii version

0.44.2

### Post editor

None

### Bug description

<img width="426" alt="image" src="https://github.com/GetPublii/Publii/assets/12690775/cc9edd95-7909-4fcf-9d29-2b831c39d534">

0.43.* and 0.44.1 have no such problem.

I can not restore any of my backup in this version, not even new created one.

btw, "Backup" is fine, only "Restore"

**Comment by Cineray** [comment]
As long as i click restore button, the Tag menu would shown same error in order to whole software crash.

**Comment by Cineray** [comment]
Hi，

May i ask why remove restore function in backup page?

That is really helpful especially when i sync everything on different OS or environment. 

I really hope it back asap.

Thank you

**Comment by bobmitro** [comment]
We had to temporarily disable this function on Windows due to a specific issue with the database library (node-sqlite3-wasm) that we're using.

The problem, identified only on Windows 11, involves the library not correctly closing the database connection. This issue is exclusive to Windows; it wasn't observed on macOS or Linux. Our efforts to resolve this, including custom builds and adjustments in the close functions, have not yet been successful. This led us to temporarily remove the direct backup restoration option for Windows versions while it remains available for macOS and Linux.

Windows users can still restore the backup using the "Install from Backup" feature on the "Add New Website" screen which is detailed described here: https://getpublii.com/docs/creating-your-website.html#creatingasitefromabackup

Additionally, we have informed the author of the node-sqlite3-wasm library about this issue, and we are actively working with them to find a resolution. We are committed to resolving this issue and hope to reinstate the direct restore function for Windows as soon as possible.
